source/_integrations/prometheus.markdown

```diff
@@ -19,6 +19,7 @@ To use the `prometheus` integration in your installation, add the following to y
 ```yaml
 # Example configuration.yaml entry
 prometheus:
+  namespace: hass
```

Comment:
We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample.

I this should have a sane default, I suggest we adjust the integration for this (although that will be quite a breaking change).

Alternatively, a more extensive example can be defined below.